prod-refractr.yml

```diff
@@ -1321,6 +1321,18 @@ refracts:
       }
     }
 
-# OPST-2405
+# OPST-2429
+- newproducts.mozilla.org/#stories:
+  - future.mozilla.org/news/
+
+- newproducts.mozilla.org/#products:
+  - future.mozilla.org/projects/
+
+- newproducts.mozilla.org/#signup:
+  - future.mozilla.org/#newsletter-section
+
```

Comment:
Question: I wanted to have the per-path redirects checked before the domain-level one. Is this is the right approach, or should i merge what's above here with what's below here?

Reply:
Hey @stevejalim! Did some trial-and-error around those refracts, I think there's no way for us to do this since refractr simply is nginx at the end, so those sources will get rendered into nginx location strings .. which don't get anything behind the `#`. Sorry!